### PR TITLE
Correctly address our users in emails.

### PR DESCRIPTION
**Pull request by MatthijsKok**

### CODE REVIEW

**Comment by martijnjanssen** [comment]
Looks good to me, haven't tested it yet though.

File: src/main/java/ch/wisv/areafiftylan/utils/mail/MailServiceImpl.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ public void sendVerificationmail(User user, String url) {
                 "Please click on the following link to complete your registration: <a href=\"" + url + "\">" + url +
                         "</a><br /><br />If the link does not work, please copy the link and" +
                         " paste it into your browser.";
-        sendMail(user.getUsername(), user.getUsername(), "Confirm your registration", message);
+        sendMail(user.getUsername(), user.getProfile().getFirstName() + " " + user.getProfile().getLastName(), "Confirm your registration", message);
```
Comment (martijnjanssen) [inline src/main/java/ch/wisv/areafiftylan/utils/mail/MailServiceImpl.java]: Like Sille said: Please note that this is not possible, the first- and lastname are not set here.

**Comment by martijnjanssen** [comment]
A helper method would be good to check whether the first and lastname are filled in, if not, use the username as name in the email.

**Comment by skamoen** [comment]
LGTM. You can rebase and merge :+1:

**Pull request merged**